client/src/components/Newsletter.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+    <b-card
+    id="newsCard"
+    title="Trending Around">
+    <hr>
+    <div class="text-center">
+          <b-spinner v-if="isloading" label="Spinning"></b-spinner>
+          <b-carousel
+      id="carousel-1"
+      :interval="4000"
+      controls
+      indicators
+      background="#ababab"
+      style="text-shadow: 1px 1px 2px #333;"
+    >
+      <b-carousel-slide v-for="news in news" >
+          <img
+          slot="img"
+          class="d-block img-fluid w-100"
+          width="1024"
```

Comment:
why have you hard coded the values

Reply:
That is because images from news api are usually from different sizes, so I have to hardcode them. :)